### DISCUSSION TITLE
Stellarium random termination

### DISCUSSION THREAD

**Discussion by blueplanetphoto** [Stellarium/stellarium#2431] [Q&A]
I'm having issues with Stellarium randomly terminating during imaging sessions. Error ID 1002, Stellarium stopped interacting with Windows and was closed. Stellarium doesn't actually close, though, it remains open but non-responsive. Connection to the telescope mount also stops. Restarting reconnects. It will also sometimes not accurately connect to the mount, showing it pointed below the horizon when it's pointed to and aligned with Polaris. Any suggestions? I just downloaded 0.22.1 but have been using 0.21.3.0. Mount is connected using EQDIR cable (EQ6RPro), EQASCOM, PHD2, APT, Windows 10 laptop. Thank you.

**Comment by alex-w** [comment]
Please check https://answers.microsoft.com/en-us/windows/forum/all/programs-stop-responding-with-application-hang/fada9d6e-7173-e011-8dfc-68b599b31bf5
